src/System.Windows.Extensions/tests/System/Media/SoundPlayerTests.cs

```diff
@@ -248,9 +248,18 @@ public void Play_NullStream_Success()
             player.Play();
         }
 
+        [Fact]
+        [OuterLoop]
+        public void Play_InvalidFile_ShortLoadTimeout_ThrowsWebException()
+        {
+            string sourceLocation = "http://google.com";
```

Comment:
Why do we need to target an external location at all?  Can you instead do something like what I did here https://github.com/dotnet/corefx/pull/34856/files#diff-b74ddc924e690e700b3aa484d4adc918L134 ?

Reply:
OK